app/styles/_uswds-theme.scss

```diff
@@ -0,0 +1,19 @@
+/*
+----------------------------------------
+USWDS with settings overrides
+----------------------------------------
+Uncomment the following lines and add a list of changed settings
+in the form $setting: value,
+----------------------------------------
+*/
+
+@use 'uswds-core' with (
+  /* Disable scary but mostly irrelevant warnings: */
+    $theme-show-compile-warnings: false,
```

Comment:
```suggestion
  $theme-show-compile-warnings: false,
```

Reply:
Oh weird. Prettier adds this indent because it follows a `/* */` style comment. I'll change the comment to `//` so that prettier stops doing that.